drop-in/src/main/java/com/adyen/checkout/dropin/ComponentParsingProvider.kt

```diff
@@ -166,11 +164,16 @@ internal fun <T : Configuration> getDefaultConfigForPaymentMethod(
     return builder.build() as T
 }
 
-internal inline fun <reified T : Configuration> getConfigurationForAction(
-    dropInConfiguration: DropInConfiguration
-): T {
-    return dropInConfiguration.getConfigurationForAction()
-        ?: ActionDelegateProvider.getDefaultConfiguration(dropInConfiguration)
+internal fun createGenericActionConfiguration(dropInConfiguration: DropInConfiguration): GenericActionConfiguration {
+    return GenericActionConfiguration.Builder(
+        dropInConfiguration.shopperLocale,
+        dropInConfiguration.environment,
+        dropInConfiguration.clientKey
+    ).apply {
```

Comment:
Well the problem is that I don't want to add a `setAvailableActionConfigs` method because `availableActionConfigs` should be an internal map only managed by the `GenericActionConfiguration`. But I needed to access it from drop-in just for that specific case. Maybe I should mark it with `@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)`? Or maybe there's a flaw in what i'm trying to do to begin with 🤔